pyqtgraph/dockarea/DockArea.py

```diff
@@ -319,7 +319,7 @@ def apoptose(self, propagate=True):
         #print "apoptose area:", self.temporary, self.topContainer, self.topContainer.count()
         if self.topContainer is None or self.topContainer.count() == 0:
             self.topContainer = None
-            if self.temporary:
+            if self.temporary and self.home:
```

Comment:
would suggest having `and self.home is not None` if we're checking for `None` and not the boolean condition of `self.home`